lib/ember-cli-rails.rb

```diff
@@ -1,4 +1,4 @@
-require "ember-cli/railtie" if defined?(Rails)
+require "ember-cli/engine" if defined?(Rails)
```

Comment:
@rwz if the railties / engine is only loaded when `Rails` constant exists, we're really dependent on Rails, not railties.

If Rails is included in the dependencies (which it should be, this being `ember-cli-rails` and all), then engines and railties come along for the ride, right?

If not, I'll gladly add it back to the gemspec